lms/djangoapps/ccx/tests/test_utils.py

```diff
@@ -185,6 +187,7 @@ def test_auto_enroll_passed_correctly(self):
 class TestEnrollEmail(ModuleStoreTestCase):
     """tests for the enroll_email function from ccx.utils
     """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
```

Comment:
Do you depend on this being a `SplitMongoModulestore`? Or could this specification of a modulestore for this test just be left out?